pkg/frontend/operations_get.go

```diff
@@ -7,17 +7,18 @@ import (
 	"encoding/json"
 	"net/http"
 
-	"github.com/gorilla/mux"
 	"github.com/sirupsen/logrus"
 
+	"github.com/Azure/ARO-RP/pkg/api"
 	"github.com/Azure/ARO-RP/pkg/frontend/middleware"
 )
 
 func (f *frontend) getOperations(w http.ResponseWriter, r *http.Request) {
 	log := r.Context().Value(middleware.ContextKeyLog).(*logrus.Entry)
-	vars := mux.Vars(r)
 
-	operations := f.apis[vars["api-version"]].OperationList
+	apiVersion := r.URL.Query().Get(api.APIVersionKey)
+
+	operations := f.apis[apiVersion].OperationList
```

Comment:
nit: since we are only using `apiVersion` in one place, we could inline its retrieval 
```suggestion
	operations := f.apis[r.URL.Query().Get(api.APIVersionKey)].OperationList
```